### ISSUE TITLE
为什么loss一直为0，精度一直为1，训练的是inception_v1

### ISSUE THREAD

**Issue by myc00** [PanJinquan/tensorflow_models_nets#23]
2019-11-27 22:04:34.371258: Step [0]  train Loss : 0.000000, training accuracy :  1
2019-11-27 22:04:57.633260: Step [0]  val Loss : 0.000000, val accuracy :  1
------save:models\best_models_0_1.0000.ckpt
2019-11-27 22:05:36.576254: Step [10]  train Loss : 0.000000, training accuracy :  1
2019-11-27 22:06:12.278251: Step [20]  train Loss : 0.000000, training accuracy :  1
2019-11-27 22:06:32.546254: Step [20]  val Loss : 0.000000, val accuracy :  1
........
